src/ng/q.js

```diff
@@ -636,6 +636,34 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
     return deferred.promise;
   }
 
+  /**
+   * @ngdoc method
+   * @name $q#race
+   * @kind function
+   *
+   * @description
+   * Returns a promise that is resolved similarly as the first promise to resolve.
+   *
+   * @param {Array.<Promise>|Object.<Promise>} promises An array or hash of promises.
+   * @returns {Promise} Returns a promise that will be resolved as soon as the first promise in
```

Comment:
I just realized that we are using `resolved` inconsistently in the docs (either as `fulfilled` or `resolved` in the sense of the spec) :unamused: (TBH, I myself thought `resolved` means `fulfilled` until now...)

I would suggest being a little more explicit:
> Returns a promise that will be settled (i.e. fulfilled or rejected)...

Don't forget to also update the description above :innocent: